site/en/guide/_mnist_tf2.ipynb

```diff
@@ -0,0 +1,316 @@
+{
```

Comment:
We need to fork everyhting for the 2.0 transition.
We're going to use a directory for that
This notebook will be the first one.

Can you move it to:

 `site/en/2/guide/tf2.ipynb`